salt/orch/removal.sls

```diff
@@ -1,23 +1,52 @@
-# must provide the node (id) to be removed in the 'target' pillar
+{#- must provide the node (id) to be removed in the 'target' pillar #}
 {%- set target = salt['pillar.get']('target') %}
 
+{#- ... and we can provide an optional replacement node #}
+{%- set replacement = salt['pillar.get']('replacement', '') %}
+
+{#- we can skip the target (for example,e when it is unresponsive)  #}
+{#- in that case, we will not perform any cleanups in the target machine #}
+{#- (but it will be removed from the kubernetes and etcd clusters anyway) #}
+{%- set skip_target = salt.caasp_pillar.get('skip', False) %}
```

Comment:
I would not add `skip` logic here... I think it's safe to assume the cluster is working fine, as a last resort there's the force removal orchestration. Nothing will set this flag on the pillar.

Reply:
I would prefer to keep it. Even if nobody uses this in upper layers, it only adds 5 `if`s and brings some nice functionality.

Reply:
I overlooked `skip_this` in https://github.com/kubic-project/salt/pull/442. IMO if it's not used it should not be there, and we should remove it from the existing code that was merged today.